common/planners/tests/it/plan_expression.rs

```diff
@@ -91,7 +91,7 @@ fn test_expression_validate() -> Result<()> {
             desc: "today-not-pass",
             expression: Expression::ScalarFunction {
                 op: "today".to_string(),
-                args: vec![col("33")],
+                args: vec![col("a")],
```

Comment:
why we need to change this test?

Reply:
Because we do not have this col in schema.